net/squid4/Portfile

```diff
@@ -133,6 +133,22 @@ if {![variant_isset gnutls]} {
     default_variants    +openssl
 }
 
+variant sslbump conflicts gnutls description "Enable SSL Bump support" {
+    default_variants-append +openssl
+    configure.args-append   --enable-ssl-crtd
+    configure.args-append   --enable-http-violations
+    configure.args-append   --enable-security-cert-validators
+    configure.args-append   --enable-security-cert-generators
+    configure.args-append   --enable-follow-x-forwarded-for
+    configure.args-append   --enable-forw-via-db
+    configure.args-append   --disable-eui
+    configure.args-append   --enable-url-rewrite-helpers    
+}
+
+variant native description "Build native arch" {
+    configure.args-delete   --disable-arch-native
```

Comment:
How much of a difference does this make in the real world?

Reply:
Don't know,  but any potential performance boost is useful.